style.css

```diff
@@ -0,0 +1,66 @@
+/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+
+header {
+
+}
+body {
+  background-color: rgb(245, 245, 245);
+  margin: 0;
```

Comment:
Nice that you applied `margin: 0;` to the body to make the content take up all the space!